### DISCUSSION TITLE
Smooth Camera Transitions

### DISCUSSION THREAD

**Discussion by Stonerhino3D** [Sixze/ALS-Refactored#420] [General]
Trying a smooth camera transition from ALS Refactored camera to spring arm camera on mount to horse or dragon ect. I used Get View Info Location and Rotation. That gets me damn close to the camera but its a little off. The camera teleports a bit during the transition. Any ideas?

**Comment by ameaninglessname** [comment]
Did you set view target to the horse or dragon?

**Comment by Stonerhino3D** [comment]
I did I was able to smooth things out by moving the camera to 0,0,0 immediately after switching view target. Its works different from spring arm to spring arm that I have done in the past but otherwise I like it.
